### DISCUSSION TITLE
Could not find a declaration file for module 'vue-i18n'

### DISCUSSION THREAD

**Discussion by crazyone2one** [intlify/vue-i18n#1380] [Q&A]
"vue": "^3.2.47",
"vue-i18n": "^9.2.2",
"vite": "^4.3.0",

when i import createI18n from vue-i18n, got an error
![image](https://user-images.githubusercontent.com/67249067/234474847-fcfbe80c-498d-407f-9d53-af2810edef28.png)
 how to fix it?  thx

**Comment by J3m5** [comment]
How did you solved this error?

**Reply by J3m5** [reply]
Updating to 9.3.0 beta solved the issue.
